repo-scripts/size-analysis/bundle-definitions/functions.json

```diff
@@ -29,4 +29,4 @@
       }
     ]
   }
-]
+]
```

Comment:
It still thinks there's a whitespace change.

Reply:
Dunno why my formatter keeps on insisting on this. Edited in vim to avoid stripping the newline.